includes/class-amp-story-post-type.php

```diff
@@ -323,6 +347,55 @@ public static function admin_enqueue_scripts( $hook ) {
 
 	}
 
+	/**
+	 * Load font from story data.
+	 *
+	 * @param WP_Post $post Post Object.
+	 */
+	public static function load_fonts( $post ) {
```

Comment:
The downside of the batching here is that `maybeEnqueueFontStyle` can potentially unnecessarily  enqueue a font twice because of the non-matching style handles.

I think it's OK for now though as we will eventually move to a more sophisticated method for loading fonts anyway. Just wanted to write it down :-)

Reply:
I could do something different in admin, so link tags are formatted with the correct id, so javascript doesn't load it. But honestly thought a couple small css calls was not a problem here.

Reply:
Let's revisit this later if needed 👍